src/AlgebraicGeometry/Schemes/Covering/Morphisms/Types.jl

```diff
@@ -37,6 +37,7 @@ mutable struct CoveringMorphism{DomainType<:Covering,
     # TODO: check domain/codomain compatibility
     # TODO: if check is true, check that all morphisms glue and that the domain patches
     # cover the basic patches of `dom`.
+    @check
```

Comment:
Is this orphaned '@check' on purpose or was it forgotten in the last pass through the file?

Reply:
the check parameter is ignored too, but it is there for consistency.